build_tools/benchmarks/comparisons/simple_commands.py

```diff
@@ -161,4 +199,22 @@ def _generate_gpu(self, driver: str):
                        self._function_input,
                        driver=driver)
     commands.append(iree)
+
+    iree_model_path = os.path.join(self._base_dir, "models", "iree", driver,
+                                   self._model_name + "_fp16.vmfb")
+    iree = IreeWrapper(self._iree_benchmark_binary_path,
+                       self._model_name + "_fp16",
+                       iree_model_path,
+                       self._function_input,
+                       driver=driver)
+    commands.append(iree)
+
+    iree_model_path = os.path.join(self._base_dir, "models", "iree", driver,
```

Comment:
Mmm, I wonder if later on we should refactor this so that we don't rely on implicit names here. E.g., the existence of X_padfuse.vmfb should not need to be assumed in this file, while we know we are generating it in the other. Down that route we probably end up with serialized execution configs, which could then also serve as means to reproduce :) (so you'd have all the flags, git revs, etc and the scripts just execute and we make it easy to reproduce results)

Reply:
As above. A reproducer would be awesome to have.